### PR TITLE
Make batch transaction for campaign emails (RR5)

### PR DESCRIPTION
**Pull request by massudaw**
We have put the batches inside transactions.

### CODE REVIEW

File: app/bundles/CampaignBundle/Model/EventModel.php [hunk 1]
```diff
@@ -957,6 +961,9 @@ public function triggerScheduledEvents(
             }
 
             // Free RAM
+
+            $this->em->flush();
+            $this->em->getConnection().commit();
```
Comment (jojomnky) [inline app/bundles/CampaignBundle/Model/EventModel.php]: ???

shouldn't this be $this->em->getConnection()->commit();

Comment (maxlawton) [inline app/bundles/CampaignBundle/Model/EventModel.php]: This should be `$this->em->getConnection()->commit();`. Has this been tested?

Comment (massudaw) [inline app/bundles/CampaignBundle/Model/EventModel.php]: You are right this is silly wrong. Our test code didn't entered this code path. I'll make the  fix and expand the testing.

**Comment by massudaw** [comment]
Closing in favor or #6

**Pull request closed without merging**